cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -497,34 +497,34 @@ subroutine dyn_prep2 (nx_block,   ny_block,   &
                             stress12_3, stress12_4, &
                             uvel_init,  vvel_init,  &
                             uvel,       vvel,       &
-                            Tbu)
+                            TbU)
 
       integer (kind=int_kind), intent(in) :: &
          nx_block, ny_block, & ! block dimensions
          ilo,ihi,jlo,jhi       ! beginning and end of physical domain
 
       integer (kind=int_kind), intent(out) :: &
          icellt  , & ! no. of cells where icetmask = 1
-         icellu      ! no. of cells where iceumask = 1
+         icellx      ! no. of cells where icexmask = 1
```

Comment:
I would probably have simply left the x's out of all these generic variables to avoid confusion with variables that use x as a direction, although this approach is nicely parallel with the t values.  Note these x's are not capitalized, while others elsewhere are.